src/doc/unstable-book/src/language-features/cfg-boolean-literals.md

```diff
@@ -0,0 +1,22 @@
+# `cfg_boolean_literals`
+
+The tracking issue for this feature is: [#00000]
+
+[#00000]: https://github.com/rust-lang/rust/issues/00000
```

Comment:
Is updating this number part of the "note to self" above?

Reply:
Yes, "and in the unstable-book".